magprime/templates/regextra.html

```diff
@@ -29,6 +29,16 @@
                 $(".badge-type-selector").parents('.form-group').hide();
             }
         {% endif %}
+
+        {% if c.AFTER_PREREG_TAKEDOWN %}
+            // MAGFest leaves prereg running after the supposed "deadline", so we want to turn off the warnings.
+            var origTogglePrices = window.togglePrices;
+            window.togglePrices = function () {
+                origTogglePrices();
+                $('.prereg-type-closing').empty();
+            };
+            togglePrices();
+        {% endif %}
```

Comment:
If we decide that this should be universal functionality instead of being MAGFest-specific then we can move it into the core repo instead of just monkeypatching like we're doing here.